cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraExecutor.java

```diff
@@ -47,128 +38,10 @@
 public class CassandraExecutor implements Executor {
```

Comment:
It still has one method that's needed the `getCassandraDaemon()`.  I'm trying to change as little as possible in this PR.  Ideally Cassandra will adopt the SDK soon and at that point the need for the CassandraExecutor will go away entirely.